lib/results.js

```diff
@@ -22,6 +22,7 @@ function Results () {
     this.pass = 0;
     this._stream = through();
     this.tests = [];
+    this._only = 0;
```

Comment:
Yes, my preference is to initialize to null. I was trying to be consistent with the rest of the code, as you had asked. ;-)